src/components/Wallet.vue

```diff
@@ -1,32 +1,40 @@
 <template>
-  <div>
-    <el-header class="el-header">
-      <div>
-        <router-link to="/wallet/transaction">Transaction</router-link>
-      </div>
-      <div>
-        <router-link to="/wallet/receive">Receive</router-link>
-      </div>
-      <div>
-        <router-link to="/wallet/send">Send</router-link>
-      </div>
-    </el-header>
-    <router-view></router-view>
-  </div>
+<div>
+  <TopBar />
+</div>
 </template>
 
 <script>
+import TopBar from '@/components/TopBar'
 export default {
   name: 'Wallet',
+  components: {
+    TopBar,
+  },
 }
 </script>
 
-<style scoped>
-.el-header {
-  font-size: 12px;
-  display: flex;
-}
-.el-header > div {
-  margin: 0 20px;
+<style lang="scss">
+@import '@/styles/_colors.scss';
+@import '@/styles/theme.scss';
+
+ .top-bar {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    border-bottom: 1px solid $grey-4;
+    .link-tab {
+      padding: 25px;
+    }
+    .link-tab a {
+      color: $grey-6;
+      font-weight: bold;
+      text-decoration: none;
+      padding: 20px;
+    &.router-link-active{
```

Comment:
```suggestion
    &.router-link-active {
```

Reply:
I will change this in the top-bar pr also